src/utils/load-plugins.ts

```diff
@@ -29,10 +37,10 @@ export default function loadPlugin([pluginPath, options]: [
   if (isSupported(pluginPath)) {
     plugin = plugins.get(pluginPath);
   } else {
-    try {
-      plugin = require(pluginPath);
-    } catch (error) {
-      plugin = require(path.join(process.cwd(), pluginPath));
+    plugin = tryRequire(pluginPath);
```

Comment:
should we log a warning if we can't load a plugin?

Reply:
now it will log when no plugin is found